### ISSUE TITLE
Standard star categories

### ISSUE THREAD

**Issue by apcooper** [desihub/desitarget#133]
In targetmask.yaml we have these three definitions for standards:
``` 
    - [STD_FSTAR,   33, "F-type standard stars", {obsconditions: DARK|GRAY}]
    - [STD_WD,      34, "White Dwarf stars", {obsconditions: DARK|GRAY}]
    - [STD_BRIGHT,  35, "F-type standard for BRIGHT conditions", {obsconditions: BRIGHT}]
```
However, `mock.build.targets_truth` only knows about `STD_FSTAR`. Does the absence of `BRIGHT` in the obsconditions for this class then means those stars aren't used as standards in bright time (I don't think FA checks obsconditions for standards, but I'm not sure). 

We should figure out if/how to define STD_BRIGHT.

Also, following some discussion of this in the MWS group, STD_WD stars can be taken from the MWS_WD catalog (`mock_wd100pc.fits`) with `(magg < 19) & (WD == 1)`.

**Comment by moustakas** [comment]
@apcooper @forero Related to this question, can you help me understand which specific MWS samples should be selected (i.e., targeting bits set) from the galaxia mocks?  In quicksurvey2016 
as near as I can tell only the ```MWS_MAIN```, ```MWS_MAIN_VERY_FAINT```, and ```MWS_WD``` bits were set.

At minimum I'll add ```MWS_NEARBY```, although here are some of the other (currently documented) possibilities:
```
    - [MWS_MAIN_LOW_GAL_B,       8, "Milky Way Survey low galactic latitude",                 {obsconditions: BRIGHT}]
    #- [MWS_WD_LOW_GAL_B,         9, "Milky Way Survey low galactic latitude",                 {obsconditions: BRIGHT}]
    #- [MWS_NEARBY_LOW_GAL_B,    10, "Milky Way Survey low galactic latitude",                 {obsconditions: BRIGHT}]

    # 16: Very faint
    - [MWS_MAIN_VERY_FAINT,     16, "Milky Way Survey additional faint filler targets",       {obsconditions: BRIGHT|GRAY|DARK}]
    #- [MWS_WD_VERY_FAINT,       17, "Milky Way Survey additional faint filler targets",       {obsconditions: BRIGHT|GRAY|DARK}]
    #- [MWS_NEARBY_VERY_FAINT,   18, "Milky Way Survey additional faint filler targets",       {obsconditions: BRIGHT|GRAY|DARK}]

    # 32: Very bright
    #- [MWS_MAIN_VERY_BRIGHT,    32, "Milky Way Survey additional very bright targets",        {obsconditions: BRIGHT|POOR|TWILIGHT12}]
    #- [MWS_WD_VERY_BRIGHT,      33, "Milky Way Survey additional very bright targets",        {obsconditions: BRIGHT|POOR|TWILIGHT12}]
    #- [MWS_NEARBY_VERY_BRIGHT,  34, "Milky Way Survey additional very bright targets",        {obsconditions: BRIGHT|POOR|TWILIGHT12}]
```
[This isn't the full list of course -- see [targetmask.yaml](https://github.com/desihub/desitarget/blob/master/py/desitarget/data/targetmask.yaml#L58).]

In quicksurvey2016 we selected the ```MWS_MAIN``` sample in the range 15<r_DECam<19 and ```MWS_MAIN_VERY_FAINT``` in the range 19<r_DECam<20.  Is this still correct?  What about the bright sample, or the selection criteria for any of the other subsamples?  And is r_DECam still the right bandpass?  In particular, I thought these selections were going to be done in the Gaia g-band, or is that just for the WDs?

**Comment by apcooper** [comment]
We currently use MWS_MAIN, MWS_MAIN_VERY_FAINT, MWS_WD and MWS_NEARBY (which was treated as MWS_WD in the datachallenge2016, but should have been separate -- this is fixed in desitarget now).

The other commented-out options were straw-man proposals that I put in when first writing MWS entries into targetmask.yaml.  It it possible that, once we have completed our analysis of the data challenge, we will want some of these more refined categories, e.g. for reobservations, low-latitude fields, fields around e.g. clusters and satellites, etc. For the time being they should probably be removed to avoid confusion. 

A related question I raised at the time (of which there are traces in the comments in targetmask.yaml) was whether these categories will always be of the form 'MWS_MAIN_VERY_BRIGHT', or if bitwise combinations will be implemented in future (e.g. MWS_MAIN & MWS_VERY_BRIGHT). @forero ?
